zerver/webhooks/zabbix/doc.md

```diff
@@ -0,0 +1,87 @@
+Receive Zabbix notifications in Zulip!
+
+1.  {!create-stream.md!}
+
+2.  {!create-a-bot-indented.md!}
+
+    Construct the URL for the {{ integration_display_name }} bot using the bot's API key and the desired stream name:
+
+    {!webhook-url.md!}
+
+    Modify the parameters of the URL above, where `api_key` is the API key of your Zulip bot, and `stream`
+    is the stream name you want the notifications sent to. If you do not specify a `stream`,
+    the bot will send notifications to the `alerts` stream which you will need to create.
```

Comment:
The whole second step can be replaced by just `{!create-bot-construct-url-indented.md!}`. :)

Reply:
The only reason I didn't set this using that is due to fact I have the default stream set to `alerts` rather than just zabbix in view.py. And the `{!create-bot-construct-url-indented.md!}` macro sets it automatically to `zabbix`. I guess its no biggy. I can just set the default stream to `zabbix` instead.

Reply:
Ahh, excellent, good to know for the future as plan to look at creating one for Ansible notifications, but will switch to `zabbix` for this one.